ledger/eval.go

```diff
@@ -77,10 +86,18 @@ func (x *roundCowBase) txnCounter() uint64 {
 	return x.txnCount
 }
 
-func (cs *roundCowState) GetAssetCreator(assetIdx basics.AssetIndex) (basics.Address, error) {
+func (cs *roundCowState) GetAssetCreator(assetIdx basics.AssetIndex) (creator basics.Address, ok bool, err error) {
 	return cs.getAssetCreator(assetIdx)
 }
 
+func (cs *roundCowState) GetAppCreator(appIdx basics.AppIndex) (creator basics.Address, ok bool, err error) {
+	return cs.getAppCreator(appIdx)
+}
+
+func (cs *roundCowState) Round() basics.Round {
```

Comment:
is it used somewhere? Seems like a leftover after introduction
```
			AppTealGlobals: AppTealGlobals{
				CurrentRound:    eval.prevHeader.Round + 1,
				LatestTimestamp: eval.prevHeader.TimeStamp,
```

Reply:
nope -- removed, thanks!